app/views/handlers/confirmation_email.py

```diff
@@ -63,16 +64,13 @@ def get_page_title(self):
         return self.page_title
 
     def _publish_fulfilment_request(self):
-        topic_id = current_app.config["EQ_SUBMISSION_CONFIRMATION_TOPIC_ID"]
```

Comment:
Reference to `EQ_SUBMISSION_CONFIRMATION_TOPIC_ID` can be removed from deployment.yaml

Reply:
Have removed this now